### DISCUSSION TITLE
JaCoCo code-coverage check error

### DISCUSSION THREAD

**Discussion by martinf-moodys** [AbsaOSS/spline-spark-agent#762] [Q&A]
In the _Add coverage to PR_ step, I get [this error](https://github.com/AbsaOSS/spline-spark-agent/actions/runs/6667834917/job/18140192981?pr=759): _HttpError: Resource not accessible by integration_.
Does anyone know how to fix?
Thanks.

**Comment by cerveada** [comment]
I guess this may be caused by you being an external contributor. It's the test coverage percentage, we can live without it this time.
